### ISSUE TITLE
Octane re-registering of model observers on every request, when used with Laravel Nova

### ISSUE THREAD

**Issue by Katzen48** [laravel/nova-issues#3447]
- Laravel Version: 8.47.0
- Nova Version: 3.26.1
- PHP Version: 8.0
- Database Driver & Version: MariaDB 10.5.6
- Operating System and Version: Windows 10
- Browser type and version: Google Chrome 91.0.4472.106

### Description:
When using Laravel Nova together with Octane, every time a new request is received, the `Nova::serving()` callback is invoked.
Registering model observers inside that callback (as stated in the nova docs) re-register them.
That creates another instance of the same observer, resulting in a growing number of registered observers, which run the operation on after another and have a heavy impact on an applications performance.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Install octane
2. Register a model observer as stated in the nova docs, that does things 
3. Run octane
4. Reload the page multiple times
5. See that the operations, like sending an event on a broadcast channel, are executed multiple times

**Comment by crynobone** [comment]
Submitted a PR but temporarily you could use the following example:

```
use App\Models\User;
use App\Observers\UserObserver;
use Illuminate\Support\Facades\Event;
use Laravel\Nova\Nova;
use Laravel\Octane\Events\RequestReceived;

Nova::serving(function () {
    User::observe(UserObserver::class);
});

Event::listen(RequestReceived::class, function () {
    User::clearBootedModels();
    User::flushEventListeners();
});
```

**Comment by crynobone** [comment]
This has been fixed in v3.27.0: https://nova.laravel.com/releases/3.27.0

```php
use App\Models\User;
use App\Observers\UserObserver;
use Illuminate\Support\Facades\Event;
use Laravel\Nova\Nova;
use Laravel\Nova\Observable;

// Before
Nova::serving(function () {
    User::observe(UserObserver::class);
});

// After
Observable::make(User:::class, UserObserver::class);
```
